packages/blade/src/components/BladeProvider/__tests__/paymentLightTheme/paymentLightTheme.web.ts

```diff
@@ -1030,14 +1030,14 @@ const paymentLightTheme: Theme = {
       xlong: 5000,
     },
     duration: {
-      quick1: 70,
-      quick2: 150,
-      quick3: 200,
-      moderate1: 250,
-      moderate2: 300,
-      gentle1: 400,
-      gentle2: 600,
-      gentle3: 900,
+      '2xquick': 70,
+      xquick: 150,
+      quick: 200,
+      moderate: 250,
+      xmoderate: 300,
```

Comment:
Okay so bit confused here. 

In quick, `x`quick is reducing to 150 from 200ms

But in moderate, `x`moderate is increased to 300 from 250ms

How is the numbers are reducing and increasing with the same naming convention applied, is it relative or what?

Reply:
is `x` here means 1x or `small` as in xxs?

Reply:
It is relative, yes.

`gentle` is `400` but "extra gentle" implies more gently which means an increase in duration which makes `xgentle` as `600`

`quick` is `200` but "extra quick" implies more quickly which means a decrease in duration which makes `xquick` as `200`

In case of moderate, there is no obvious to decide what would be `xmoderate` - assumption is that it means "more moderate" similar to "more gentle" & "more quick" which would make `xmoderate` a value thats a little more than moderate. Hence `xmoderate` as `300`. 

We could think in reverse for moderate and make it lesser but that's just a hard call we need to make on which direction to take it 😓